### PR TITLE
OCM: provider-credential-controller - Enable DCO plugin and run -short sonar

### PR DESCRIPTION
**Pull request by jnpacker**
Signed off by @jnpacker 

* Add -short so that I can leave the scale *_test.go files in the repository.
* Add DCO plugin support.

### CODE REVIEW

File: ci-operator/jobs/open-cluster-management/provider-credential-controller/open-cluster-management-provider-credential-controller-main-postsubmits.yaml [hunk 1]
```diff
@@ -101,12 +101,9 @@ postsubmits:
...
     spec:
       containers:
-      - args:
-        - -f
-        - Makefile.prow
-        - sonar/go/prow
-        command:
-        - make
+      - command:
+        - export SONAR_GO_TEST_ARGS="-short ./..."
```
Comment (chrisahl) [inline ci-operator/jobs/open-cluster-management/provider-credential-controller/open-cluster-management-provider-credential-controller-main-postsubmits.yaml:105]: Is it `./...` (three dots?)

Comment (jnpacker) [reply ci-operator/jobs/open-cluster-management/provider-credential-controller/open-cluster-management-provider-credential-controller-main-postsubmits.yaml:105]: Yep. That's what I saw in other examples and what @ray-harris provided in slack.

**Review by alvaroaleman — APPROVED** [review]
/label tide/merge-method-squash

**Pull request merged**